perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/pwdmgr/EinfraPasswordManagerModuleTest.java

```diff
@@ -16,6 +18,9 @@
 
 	private EinfraPasswordManagerModule module;
 
+	private final int     randomPasswordLength = 12;
+	private final Pattern EinfraPasswordContainsNotAllowedChars = Pattern.compile(".*[^ABCDEFGHJKLMNPQRSTUVWXabcdefghjkmnpqrstuvwx23456789,.].*");
```

Comment:
In our code, we don't align the variable names, please use this formatting instead:
```suggestion
	private final int randomPasswordLength = 12;
	private final Pattern EinfraPasswordContainsNotAllowedChars = Pattern.compile(".*[^ABCDEFGHJKLMNPQRSTUVWXabcdefghjkmnpqrstuvwx23456789,.].*");
```